flake.lock

```diff
@@ -311,11 +311,11 @@
         "yants": "yants"
       },
       "locked": {
-        "lastModified": 1692005050,
-        "narHash": "sha256-bFhmzmenx3Y/0eH1qAAZGrmFc3siz0BiAJRptdt2QIQ=",
+        "lastModified": 1692627026,
```

Comment:
Why are there changes to the `flake.lock` if there's no package changes in the scope of this commit?